6-infrastructure-configuration-management/6.1.4-terraform-providers.md

```diff
@@ -14,49 +14,199 @@ Throughout this section it will be helpful to refer to the [Custom Framework Pro
+
+5. Now that you have your repo generated we now need to invite the rest of your team members to the repository. 
+    
+    Have the owner of the repository go into the repository `Settings` tab and click on `Collaborators`
+
+    ![terraform provider settings tab](img6/settings_tab.png ':size=906x80')
+    
+    Here you can see a button called `Add people`. Click on to get a pop-out to start sending invitiations to your team-members to collaborate for the terraform provider.
+    
+    ![terraform provider collab tab](img6/collab_tab.png ':size=906x406')
+
 ### Exercise 1: Provider Boilerplate
 
 This exercise will walk you through setting up the boilerplate for a Terraform provider and ensure you can test it locally.
 
-1. Clone the [terraform-provider-scaffolding](https://github.com/hashicorp/terraform-provider-scaffolding-framework) repo.
+1. Clone the repository made in the Setup section.
 
 2. Change the provider name to `devops-bootcamp` and configure the provider server address as `liatr.io/terraform/devops-bootcamp`
 
 3. Test that the provider executes correctly by running `go run main.go`. You should see the following output:
-  ```
-  This binary is a plugin. These are not meant to be executed directly.
-  Please execute the program that consumes these plugins, which will
-  load any plugins automatically
-  exit status 1
-  ```
+   ``` 
+   This binary is a plugin. These are not meant to be executed directly.
+   Please execute the program that consumes these plugins, which will
+   load any plugins automatically
+   exit status 1
+   ```
 
 4. Build a binary for your provider `go build -o terraform-provider-devops-bootcamp`
 
 5. Edit `~/.terraformrc` and add the follow dev override to use the local version of your provider:
-  ```
-  provider_installation {
+    ```go
+    provider_installation {
 
-    dev_overrides {
-        "liatr.io/terraform/devops-bootcamp" = "PATH_TO_PROVIDER"
-    }
+      dev_overrides {
+          "liatr.io/terraform/devops-bootcamp" = "PATH_TO_PROVIDER"
+      }
 
-    direct {}
-  }
-  ```
+      direct {}
+    }
+    ```
 
 6. Create a Terraform module with the following configuration to test your provider:
-  ```
-  terraform {
-    required_providers {
-      devops-bootcamp = {
-        source = "liatr.io/terraform/devops-bootcamp"
+    ```go
+    terraform {
+      required_providers {
+        devops-bootcamp = {
+          source = "liatr.io/terraform/devops-bootcamp"
+        }
       }
     }
-  }
 
-  provider "devops-bootcamp" {
-    # example configuration here
-  }
-  ```
+    provider "devops-bootcamp" {
+      # example configuration here
+    }
+    ```
 
 7. Run `terraform plan` to ensure Terraform can find and use your provider.
+
+### API Overview
+
+This section of the boot camp will focus on using an ephemeral API
+([API ReadMe](https://github.com/liatrio/devops-bootcamp/blob/a331c9598fdd5187eb324a9b54578e45b7a5dfb9/examples/ch6/devops-api/readMe.md))
```

Comment:
Is this link correct? It probably shouldn't link to a specific commit, but if it does need to, the readme appears to be empty in this commit.

Reply:
Fixed empty document link with this [commit](https://github.com/liatrio/devops-bootcamp/pull/214/commits/5b1a5880be11f002a41753756d4fd3b3aa8414e5). Still links to a specific commit as a permalink for the API readme.